src/systems/tips.js

```diff
@@ -82,263 +37,62 @@ const tipPlatform = () => {
-    if (
-      !(
-        userinput.activeSets.includes(sets.rightCursorHoldingInteractable) ||
-        userinput.activeSets.includes(sets.leftCursorHoldingInteractable)
-      )
-    )
-      return INVALID;
-    if (
-      userinput.get(paths.actions.cursor.right.scaleGrabbedGrabbable) ||
-      userinput.get(paths.actions.cursor.left.scaleGrabbedGrabbable)
-    )
-      return FINISH;
-    return VALID;
-  },
-  pen_color: function(userinput) {
-    if (
-      !(
-        userinput.activeSets.includes(sets.rightCursorHoldingPen) ||
-        userinput.activeSets.includes(sets.leftCursorHoldingPen)
-      )
-    )
-      return INVALID;
-    if (
-      userinput.get(paths.actions.cursor.right.penNextColor) ||
-      userinput.get(paths.actions.cursor.right.penPrevColor)
-    ) {
-      return FINISH;
-    }
-    if (
-      userinput.get(paths.actions.cursor.left.penNextColor) ||
-      userinput.get(paths.actions.cursor.left.penPrevColor)
-    ) {
-      return FINISH;
-    }
-    return VALID;
-  },
-  pen_size: function(userinput) {
-    if (
-      !(
-        userinput.activeSets.includes(sets.rightCursorHoldingPen) ||
-        userinput.activeSets.includes(sets.leftCursorHoldingPen)
-      )
-    )
-      return INVALID;
-    if (userinput.get(paths.actions.cursor.right.scalePenTip) || userinput.get(paths.actions.cursor.left.scalePenTip))
-      return FINISH;
-    return VALID;
-  },
-  pen_mode: function(userinput) {
-    if (
-      !userinput.activeSets.includes(sets.rightCursorHoldingPen) &&
-      !userinput.activeSets.includes(sets.leftCursorHoldingPen)
-    )
-      return INVALID;
-    return VALID;
-  },
-  freeze_mode: function(userinput, scene) {
-    return scene.is("frozen") ? VALID : INVALID;
-  },
-  video_share_mode: function(userinput, scene) {
-    return scene.is("sharing_video") ? VALID : INVALID;
-  },
-  mute_mode: function(userinput, scene) {
-    return scene.is("muted") ? VALID : INVALID;
-  },
-  feedback: function(/*userinput, scene, mediaCounter, store*/) {
-    /*if (configs.feature("show_feedback_ui") && store && store.state.activity.entryCount >= NUM_ENTRIES_FOR_FEEDBACK_TIP)
-      return VALID;*/
-    return INVALID;
   }
 };
 
 AFRAME.registerSystem("tips", {
   init: function() {
-    this.activeTips = {};
-    this._finishedScopes = {};
+    this.activeTip = null;
     this._performStep = this._performStep.bind(this);
 
     if (localStorage.getItem(LOCAL_STORAGE_KEY) === null) {
       localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify({}));
     }
   },
 
+  resetTips: function() {
+    localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify({}));
+    localStorageCache = null;
+    window.APP.store.resetTipActivityFlags();
+    window.APP.store.resetConfirmedBroadcastedRooms();
+    this.activeTip = null;
+    finished = false;
+  },
+
+  skipTips: function() {
+    for (let i = 0; i < platformTips.length; i++) {
```

Comment:
Seems like the skip button should only skip a single tip - not dismiss all of them.

Reply:
I'm pretty much keeping the existing behavior here. Skipping all the tips is a pretty common action on these onboarding flows. I think we'll look into onboarding in detail with the design team soon.